app/assets/javascripts/discourse/app/routes/build-group-messages-route.js

```diff
@@ -31,10 +31,22 @@ export default (type) => {
         hideCategory: true,
         showPosters: true,
       });
+
+      const currentUser = this.currentUser;
+      this.searchService.set("searchContext", {
+        type: "private_messages",
+        id: currentUser.get("username_lower"),
+        user: currentUser,
+      });
     },
 
     _isArchive() {
       return type === "archive";
     },
+
+    deactivate() {
+      // Remove the search context
```

Comment:
```suggestion
```

I don't think there is a need for this comment. The code itself is pretty clear about what it is doing.